polylaue/ui/region_mapping/dialog.py

```diff
@@ -211,6 +216,39 @@ def showEvent(self, event: QShowEvent) -> None:
 
         return super().showEvent(event)
 
+    def keyPressEvent(self, event: QEvent):
+        """Override the key press event to navigate between scan numbers"""
+
+        def shift_position(i, j):
+            self.shift_scan_position.emit(i, j)
+            event.accept()
+
+        def shift_scan_number(i):
+            self.shift_scan_number.emit(i)
+            event.accept()
+
+        match event.key():
+            case Key.Key_Right:
+                # Move right one column
+                return shift_position(0, 1)
+            case Key.Key_Left:
+                # Move left one column
+                return shift_position(0, -1)
+            case Key.Key_Down:
+                # Move down one row
+                return shift_position(1, 0)
+            case Key.Key_Up:
+                # Move up one row
+                return shift_position(-1, 0)
+            case Key.Key_PageUp:
+                # Move up one scan
+                return shift_scan_number(1)
+            case Key.Key_PageDown:
+                # Move down one scan
+                return shift_scan_number(-1)
+
+        return super().keyPressEvent(event)
```

Comment:
This looks fine for now. At some point, we might want to merge this one with the one in the image view and re-use it for all dialogs in the application, if possible.